queries/unison/highlights.scm

```diff
@@ -0,0 +1,75 @@
+;; Primitives
+(comment) @comment
+(nat) @constant.numeric
+(unit) @constant.builtin
+(literal_char) @constant.character
+(literal_text) @string
+(literal_boolean) @constant.builtin.boolean
+
+;; Keywords
+[
+  (kw_forall)
+  (unique_kw)
+  (type_kw)
+  (kw_equals)
+  (do)
+] @keyword
+
+(kw_let) @keyword.function
+(structural_kw) @keyword.storage.type
+(unique) @keyword.storage.modifier
+
+[ 
+  (type_constructor)
+] @constructor
+
+(doc_block) @comment
+
+
+[
+  (operator)
+  (pipe) 
+  (arrow_symbol) 
+  (">")
+  (or)
+] @operator
+
+[
+  "if"
+  "else"
+  "then"
+  (match)
+  (with)
+  (cases)
```

Comment:
Can't the match, with, cases string themselves be used instead of nodes?

Reply:
I tried with these as strings and it breaks, though I'm having trouble seeing the details of the error other than it just gets angry.